pkg/registers/txt.go

```diff
@@ -29,15 +30,57 @@ type PhysicalMemoryReader interface {
 	ReadPhysBuf(addr int64, buf []byte) error
 }
 
-//FetchTXTConfigSpace returns a raw copy of the TXT config space
-func FetchTXTConfigSpace(txtAPI PhysicalMemoryReader) (TXTConfigSpace, error) {
+// FetchTXTConfigSpaceRaw returns a raw copy of the TXT config space
+//
+// Warning, this function may trigger undocumented registers which affects
+// other registers or anything else.
+func FetchTXTConfigSpaceRaw(mem PhysicalMemoryReader) (TXTConfigSpace, error) {
 	data := make([]byte, TxtPublicSpaceSize)
-	if err := txtAPI.ReadPhysBuf(TxtPublicSpace, data); err != nil {
+	if err := mem.ReadPhysBuf(TxtPublicSpace, data); err != nil {
 		return nil, err
 	}
 	return data, nil
 }
 
+// FetchTXTConfigSpace is a deprecated alias for FetchTXTConfigSpaceRaw.
+// DEPRECATED: The function was renamed to FetchTXTConfigSpaceRaw.
+func FetchTXTConfigSpace(mem PhysicalMemoryReader) (TXTConfigSpace, error) {
+	return FetchTXTConfigSpaceRaw(mem)
+}
+
+var (
+	physMemDenyList = pkgbytes.Ranges{
+		{
+			// This is an undocumented register which triggers ACM_POLICY_STATUS
+			// corruption.
+			Offset: 0xFED30370,
+			Length: 8,
+		},
+	}
+)
+
+// FetchTXTConfigSpaceSafe returns a filtered raw copy of the TXT config space,
+// it excludes registers, which is not supposed to be read (in contrast
+// to FetchTXTConfigSpaceSafe).
```

Comment:
nit: in contrast to FetchTXTConfigSpaceRaw

Reply:
Cannot change without resetting the approvals :(

Reply:
Will make an additional PR.

Reply:
https://github.com/9elements/converged-security-suite/pull/253/files